pages/all/[id].jsx

```diff
@@ -1,101 +1,127 @@
+  let japaneseFirstCommentReplies = [];
+  if (firstCommentReplies) {
+    japaneseFirstCommentReplies = await Promise.all(
+      firstCommentReplies.map((fisrtCommentReply) =>
+        translateCommentDetail(fisrtCommentReply, JA)
+      )
     );
   }
+  // console.log(japaneseFirstCommentReplies);
 
   return {
-    props: { story, topComment, topCommentReplies },
+    props: {
+      japaneseStoryDetail,
+      japaneseFirstCommentDetail,
+      japaneseFirstCommentReplies,
+    },
     revalidate: 10,
   };
 }
 
 export async function getStaticPaths() {
-  // 1.This is top 3 story ids.
-  const resOne = await fetch(
-    `https://hacker-news.firebaseio.com/v0/topstories.json?print=pretty&limitToFirst=10&orderBy="$key"`
-  );
-  const topstories = await resOne.json();
+  let topStoriesIds = [];
+  try {
+    const getTopStoriesIdsRes = await fetch(
+      `https://hacker-news.firebaseio.com/v0/topstories.json?print=pretty&limitToFirst=10&orderBy="$key"`
+    );
 
-  const paths = topstories.map((topstory) => ({
-    params: { id: topstory.toString() },
+    if (!getTopStoriesIdsRes.ok) {
+      return {
+        notFound: true,
+        revalidate: 10,
+      };
+    }
+    topStoriesIds = await getTopStoriesIdsRes.json();
+  } catch (error) {
+    return {
+      notFound: true,
+      revalidate: 10,
+    };
+  }
+
+  const paths = topStoriesIds.map((topStoriesId) => ({
+    params: { id: topStoriesId.toString() },
   }));
-  console.log(paths);
+  // console.log(paths);
   return {
     paths,
     fallback: false,
   };
 }
 
-const DetailPage = ({ story, topComment, topCommentReplies }) => {
+const DetailPage = ({
+  japaneseStoryDetail,
+  japaneseFirstCommentDetail,
+  japaneseFirstCommentReplies,
+}) => {
   return (
     <div>
       <PageTitle />
       <div className={"main_container"}>
         <div className="detail_article_title_container">
-          <DetailArticleTitle detailArticleTitle={story.title} />
+          <DetailArticleTitle detailArticleTitle={japaneseStoryDetail.title} />
         </div>
         <div className="article_text_container">
           <DetailArticleCategoryTitle
-            detailArticleCategoryTitle={"Article Summary"}
+            detailArticleCategoryTitle={"Story URL"}
           />
           <div className="main_text-container">
-            <DetailArticleText
-              detailArticleText={
-                "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Tempor nec feugiat nisl pretium. Purus non enim praesent elementum facilisis. Ultrices sagittis orci a scelerisque purus semper eget duis. At lectus urna duis convallis convallis tellus. Gravida in fermentum et sollicitudin ac orci phasellus egestas. Ut sem nulla pharetra diam sit. Id donec ultrices tincidunt."
-              }
-            />
+            <DetailArticleUrl DetailArticleUrl={japaneseStoryDetail.url} />
```

Comment:
minor: the name of the prop should be camelcase.

Reply:
I understand. 
Changed the name -->[link](https://github.com/yuki-o1o5/hacker-news-in-japanese/commit/4b98b3a296d89d7d1121ac4d40a99c3ed1adf814)